lib/json_api_resource/handlers/find_handler.rb

```diff
@@ -0,0 +1,19 @@
+module JsonApiResource
+  module Handlers
+    class FindHandler
+      attr_accessor :result
+
+      def initialize(results)
+        simplify!(results)
+      end
+
+      def simplify!(results)
+        result = results.first
+
+        result.meta = results.meta
+
+        result.linked_data = results.try(:linked_data)
+      end
```

Comment:
what is this method doing? simplify seems a bit generic.  This seems like its"

1. getting the first result
2. setting meta to it
3. appending linked data

maybe `build_from_first` or `build_first`?